_posts/2023-11-22-unpacking-langstream.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Unpacking LangStream's Event-Driven Architecture"
+categories:
+image: /image/blog/article-15-icon.png
+author_staff_member: "LangStream"
```

Comment:
Can you create a langstream.md in the _staff_members directory and reference it here (as `langstream`) with the site logo as the image? That way the "LangStream" author will render properly.